docs/source/user-guide/tutorials/build-pkgs-skeleton.rst

```diff
@@ -7,6 +7,11 @@ Building conda packages with conda skeleton
    :local:
    :depth: 1
 
+.. warning::
+   For a better experience with building conda packages from PyPI pacakages, we recommend the usage
```

Comment:
```suggestion
   For a better experience with building conda packages from PyPI packages, we recommend the usage
```

Reply:
I would say "We recommend Grayskull, a newer alternative to conda-skeleton, to generate conda recipes for PyPI packages."